src/main/frontend/fs/sync.cljs

```diff
@@ -2980,11 +2976,44 @@
 
     (reset! current-sm-graph-uuid nil)))
 
+(defn <sync-local->remote []
+  (go
+    (when-let [_sm ^SyncManager (state/get-file-sync-manager (state/get-current-file-sync-graph-uuid))]
+      (offer! immediately-local->remote-chan true))))
+
 (defn sync-need-password!
   []
   (when-let [sm ^SyncManager (state/get-file-sync-manager (state/get-current-file-sync-graph-uuid))]
     (.need-password sm)))
 
+(def app-state-changed-cursor (rum/cursor state/state :mobile/app-state-change))
+
+(def finished-local->remote-chan (chan 1))
```

Comment:
why not just sub sync-events-publication for :finished-local->remote

Reply:
hmm, it's actually sub the event [here](https://github.com/logseq/logseq/pull/7292/files#diff-8c983af40a8bd3dde30495140e8989f44c6a3423b49e3347f7c490834c491690R206)